### PR TITLE
Added a way of setting an implicit set to the compiler options.

### PR DESCRIPTION
**Pull request by AlvaroBarua**

### CODE REVIEW

**Comment by AlvaroBarua** [comment]
@googlebot I signed it!

**Comment by zoddicus** [comment]
I just noticed that this never was reviewed.

@dneto0 when you get back to the office can you take a look at it?

**Comment by AlvaroBarua** [comment]
Hi, just remembered I added this pull request. I was wondering if it was reviewed?

File: libshaderc_util/include/libshaderc_util/compiler.h [hunk 1]
```diff
@@ -328,6 +328,23 @@ class Compiler {
...
+    for (auto stage : stages()) {
+      SetImplicitSetForStage(stage, set);
+    }
+  }
+
+  // Sets an implicit set for the given GLSL shader stage if no set has been provided in
+  // the qualifier
+  void SetImplicitSetForStage(Stage stage, uint32_t set)
+  {
+    hlsl_explicit_bindings_[static_cast<int>(stage)].clear();
```
Comment (dneto0) [inline libshaderc_util/include/libshaderc_util/compiler.h:344]: This interferes with the operation of SetHlslRegisterSetAndBindingForStage

Also, glslang only operates on these options in triples.
See https://github.com/KhronosGroup/glslang/blob/b2d2c9dd3d77c6adadff37309aa58a181bafc5f9/glslang/HLSL/hlslParseHelper.cpp#L6388

**Review by dneto0 — CHANGES_REQUESTED** [review]
I am skeptical that this works.

Also, there are no tests.

**Pull request closed without merging**